<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Qcentris-396" failures="1" tests="2" name="LoginErrorValidation" time="4.278" errors="0" timestamp="2025-01-05T11:50:19 EET">
  <testcase classname="tests.ErrorValidationTest" name="testInvalidLoginLandingPage" time="0.463">
    <skipped/>
  </testcase> <!-- testInvalidLoginLandingPage -->
  <testcase classname="tests.ErrorValidationTest" name="testInvalidLoginLandingPage" time="0.368">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at tests.ErrorValidationTest.testInvalidLoginLandingPage(ErrorValidationTest.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- testInvalidLoginLandingPage -->
</testsuite> <!-- LoginErrorValidation -->
